File: benchmarks/looping/loop.py
import time

def main():
    start = time.time() * 1000
    total = 0
    i = 0
    while i < 10000000:
        total += i
        i += 1
    end = time.time() * 1000
    print(f"Python: {int(end - start)} ms (Result: {total})")

main()

File: benchmarks/looping/loop.cpp
#include <iostream>
#include <chrono>

int main() {
    auto start = std::chrono::high_resolution_clock::now();
    long long total = 0;
    for (int i = 0; i < 10000000; i++) {
        total += i;
    }
    auto end = std::chrono::high_resolution_clock::now();
    auto duration = std::chrono::duration_cast<std::chrono::milliseconds>(end - start);
    std::cout << "C++: " << duration.count() << " ms (Result: " << total << ")" << std::endl;
    return 0;
}
